src/ReplayCommand.cc

```diff
@@ -69,7 +69,10 @@ ReplayCommand ReplayCommand::singleton(
     "  --stats=<N>                display brief stats every N steps (eg 10000).\n"
     "  --serve-files              Serve all files from the trace rather than\n"
     "                             assuming they exist on disk. Debugging will\n"
-    "                             be slower, but be able to tolerate missing files\n");
+    "                             be slower, but be able to tolerate missing files\n"
+    "  --target-wine              Tell gdb to use a library list supplied by rr.\n"
+    "                             This requires to jump after wine finished process\n"
+    "                             creation by adding option -g.\n");
```

Comment:
Why do users have to use `-g` like this?

Could we automatically detect WINE instead?

Reply:
> Why do users have to use `-g` like this?

I did not get past this "Program stopped" message.

```
benutzer@debian:~/wine/drive_c/REVOLT$ rr replay wine-1
...
(rr) cont
Continuing.

Program stopped.
0x70000002 in syscall_traced ()
(rr) bt
#0  0x70000002 in syscall_traced ()
#1  0xb7f7e183 in _raw_syscall () at /home/bernhard/data/entwicklung/2021/rr/2021-04-09/rr/src/preload/raw_syscall.S:34
#2  0xb7f7c194 in traced_raw_syscall (call=0x681fffd8) at /home/bernhard/data/entwicklung/2021/rr/2021-04-09/rr/src/preload/syscallbuf.c:278
#3  syscall_hook_internal (call=0x681fffd8) at /home/bernhard/data/entwicklung/2021/rr/2021-04-09/rr/src/preload/syscallbuf.c:3360
#4  syscall_hook (call=<optimized out>) at /home/bernhard/data/entwicklung/2021/rr/2021-04-09/rr/src/preload/syscallbuf.c:3394
#5  0xb7f793e4 in _syscall_hook_trampoline () at /home/bernhard/data/entwicklung/2021/rr/2021-04-09/rr/src/preload/syscall_hook.S:139
#6  0xb7f7944d in _syscall_hook_trampoline_90_90_90 () at /home/bernhard/data/entwicklung/2021/rr/2021-04-09/rr/src/preload/syscall_hook.S:223
#7  0x6ffff005 in __kernel_vsyscall ()
#8  0xb7e4fc79 in execve () at ../sysdeps/unix/syscall-template.S:120
#9  0x00417f51 in ?? ()
#10 0x004184a9 in ?? ()
#11 0x00417edb in ?? ()
#12 0x00417270 in ?? ()
#13 0x00417963 in ?? ()
#14 0x00416650 in ?? ()
#15 0x0041e262 in ?? ()
#16 0x0041478f in ?? ()
#17 0xb7da3e46 in __libc_start_main (main=0x414590, argc=3, argv=0xbfc8b0d4, init=0x427b20, fini=0x427b80, rtld_fini=0xb7fa0230 <_dl_fini>, stack_end=0xbfc8b0cc) at ../csu/libc-start.c:308
#18 0x00414811 in ?? ()
(rr) when 
Current event: 1050
(rr) info inferior
  Num  Description       Connection                         Executable        
* 1    process 2011      1 (extended-remote 127.0.0.1:2235) /home/benutzer/.local/share/rr/wine-1/mmap_pack_213_dash 
(rr) kill
Kill the program being debugged? (y or n) y
[Inferior 1 (process 2011) killed]
(rr) q
benutzer@debian:~/wine/drive_c/REVOLT$ rr replay -g 1060 wine-1
...
Remote debugging using 127.0.0.1:2250

--------------------------------------------------
 ---> Reached target process 2011 at event 1061.
--------------------------------------------------
Reading symbols from /lib/ld-linux.so.2...
Reading symbols from /usr/lib/debug/.build-id/77/17f60b9dcc4447c397f1d52e9845e938cdca62.debug...
BFD: warning: system-supplied DSO at 0x6fffd000 has a section extending past end of file
0xb7fd8117 in get_extended_indices (cpu_features=<optimized out>) at ../sysdeps/x86/cpu-features.c:49
49      ../sysdeps/x86/cpu-features.c: Datei oder Verzeichnis nicht gefunden.
(rr) info inferior
  Num  Description       Connection                         Executable        
* 1    process 2011      1 (extended-remote 127.0.0.1:2250) /home/benutzer/.local/share/rr/wine-1/mmap_pack_165_wine 
(rr)
```